selfdrive/ui/sunnypilot/qt/offroad/settings/vehicle_panel.cc

```diff
@@ -20,11 +20,101 @@ VehiclePanel::VehiclePanel(QWidget *parent) : QFrame(parent) {
   platformSelector = new PlatformSelector();
   list->addItem(platformSelector);
 
+//   hkgtuningToggle = new ParamControl("HKGtuning", (add all toggle parameters here), this);
+
+  QObject::connect(uiState(), &UIState::offroadTransition, [=](bool offroad) {
+    isOnroad = !offroad;
+    updateCarToggles();
+  });
+
   ScrollViewSP *scroller = new ScrollViewSP(list, this);
   vlayout->addWidget(scroller);
 
   QObject::connect(uiState(), &UIState::offroadTransition, platformSelector, &PlatformSelector::refresh);
 
+  QObject::connect(platformSelector, &PlatformSelector::clicked, [=]() {
+    QTimer::singleShot(100, this, &VehiclePanel::updateCarToggles);
+  });
+
+  QTimer *timer = new QTimer(this);
+  QObject::connect(timer, &QTimer::timeout, this, &VehiclePanel::updateCarToggles);
+  timer->start(1000);
```

Comment:
The `QTimer` used here along with the `platformSelector` button currently delays for 100 ms after being clicked, then invokes `updateCarToggles` every 1 second. Do we have to refresh the panel continuously, even with no user interaction?